### PR TITLE
Update turkish translate

### PR DESCRIPTION
**Pull request by ardadem**

### CODE REVIEW

**Comment by def-** [comment]
@Learath2 ?

File: data/languages/turkish.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #	Learath2		2011-08-21 00:12:50
 #	Learath2		2011-09-07 15:59:20
 #	Learath2		2012-01-01 22:54:29
+#       ardadem                 2020-08-20 00:00:00
```
Comment (Learath2) [inline data/languages/turkish.txt:9]: Whitespace?

**Comment by Learath2** [comment]
LGTM, can merge after he fixes the alignment.

**Comment by ardadem** [comment]
Done

**Comment by Learath2** [comment]
Thanks

bors r+

**Comment by Learath2** [comment]
Uuuh, you might want to rebase that

**Pull request merged**